doc/src/upgrade.md

```diff
@@ -15,6 +15,29 @@ Contact our [support](/platform/index.html#support) for upgrade assistance.
 
 ## Overview
 
+- New roles:
+    - {ref}`postgresql18 <nixos-upgrade-postgresql>`
+- Removed roles:
+    - `mysql57`
+    - `percona83`
+    - `postgresql13`
+- Roles affected by significant breaking changes:
+    - {ref}`webgateway <nixos-upgrade-webgateway>`
+    - {ref}`mailserver <nixos-upgrade-mail>`
+    - {ref}`mysql57 percona83 <nixos-upgrade-percona>`
```

Comment:
This is duplicating the *Removed roles* mention above.

If this is done deliberately to raise awareness on the required upgrade steps for users of mysql57 and percona83, this is a bit of splitting hair on whether *removing a role* is also considered a *significant breaking change*.

I advocate for putting the reference to the upgrade details section on the *Removed roles* items and removing it from the 2nd section.